unstable/lens/struct-list.scrbl

```diff
@@ -0,0 +1,23 @@
+#lang scribble/manual
+
+@(require lens/private/doc-util/main)
+
+@title{Converting between structs and lists}
+
+@deftogether[[
+  @defform[(struct->list-lens struct-id)]
+  @defform[(list->struct-lens struct-id)]]]{
+Lenses that convert between structs and lists.
```

Comment:
No, because it actually works fine on opaque structs. The examples show that when it converts from structs to lists, but the printing isn't ideal for showing how it converts from lists to structs. So it actually does the right thing even when it's opaque, so it has no need for an `on-opaque` argument.